docker/aws/run_ssm_command.py

```diff
@@ -0,0 +1,14 @@
+import sys
+from sessions import get_session
+
+account_number = sys.argv[1]
+stage = sys.argv[2]
+instance_name = sys.argv[3]
+command_name = sys.argv[4]
+
+boto_session = get_session(account_number, "TDRJenkinsRunDocumentRole" + stage.capitalize())
+client = boto_session.client('ssm', region_name='eu-west-2')
+
+parameter = client.send_command(Targets=[{'Key': 'tag:Name','Values': [instance_name]}], DocumentName=command_name)
+
+print(parameter)
```

Comment:
Is this print statement needed? Could be a problem if the parameter contains sensitive info

Reply:
It was just so there was some output in Jenkins that we can see to know it's worked. I know it would fail if it hadn't worked but it's nice to have something to see. You're right though, it doesn't need to be the entire response so I've changed it to something more generic.